src/niweb/niweb/templates/base.html

```diff
@@ -142,20 +142,24 @@
                         <li class="divider"></li>
                         <li class="nav-header"><i class="icon-list"></i> Browse types</li>
                         {% type_menu %}
-                        <li class="divider"></li>
-                        <li class="nav-header"><i class="icon-briefcase"></i> Reports</li>
-                        <li><a href="/reports/hosts/">Host reports</a></li>
-                        <li><a href="/reports/unique-ids/">Unique IDs</a></li>
-                        <li class="divider"></li>
-                        <li class="nav-header"><i class="icon-map-marker"></i> Maps</li>
-                        <li><a href="/gmaps/sites/">Site map</a></li>
-                        <li><a href="/gmaps/optical-nodes/">Optical node map</a></li>
+                        {% ifequal menu_mode 'ni' %}

```

Comment:
Hmm what does menu_mode 'ni' really mean?

Reply:
This just hides the NI specific menu items when in "sri"-mode. menu_mode ni is the default in global preferences.